### DISCUSSION TITLE
Wondering if this works on 7800X3D?

### DISCUSSION THREAD

**Discussion by dtr125** [svenlange2/Ryzen-5800x3d-linux-undervolting#10] [Q&A]
As per the title, wondering if anyone has tried it on the 7800X3D yet to see if it works on there too?

**Comment by svenlange2** [comment]
7800X3D is also a single CCX (core complex), so i guess very similar CPUs. So what is needed is to find out what the addresses for the offsets are in the Ryzen SMU memory space. Another story is if this all is worth it as I have read there are really small gains to be had in case of 7800X3D.
